plugin/web/components/common/Icon/index.tsx

```diff
@@ -0,0 +1,36 @@
+import { styled } from "@web/theme";
+import { memo } from "react";
+
+import icons from "./icons";
+
+type Props = {
+  icon: string;
+  size?: string | number;
+};
+
+type Icons = keyof typeof icons;
+
+const Icon: React.FC<Props> = ({ icon, size }) => {
+  const sizeStr = typeof size === "number" ? `${size}px` : size;
+  const Iconele = icons[icon as Icons];
+
+  return (
+    <Wrapper size={sizeStr}>
+      <Iconele />
+    </Wrapper>
+  );
+};
+
+const Wrapper = styled.div<{ size: string }>`
+  font-size: 0;
+  display: inline-block;
+  width: ${({ size }: any) => size};
+  height: ${({ size }: any) => size};
```

Comment:
```suggestion
  width: ${({ size }) => size};
  height: ${({ size }) => size};
```

Reply:
I couldn't remove the any , it keeps showing the error "Binding element 'size' implicitly has an 'any' type."

Reply:
[pr-fix1](https://github.com/eukarya-inc/reearth-plateauview/pull/30/commits/4368dbe8b4bb396983a7cc21a87dc8497771a482)